layout/explore-detail/explore-detail-actions.js

```diff
@@ -93,12 +95,8 @@ export const fetchTags = createThunkAction('WIDGET-DETAIL/fetchTags', () => (dis
 // COUNT VIEW
 export const setCountView = createThunkAction('WIDGET-DETAIL/setCountView', () => (dispatch, getState) => {
   const { exploreDetail, user } = getState();
-
-  const service = new GraphService();
-
   if (!user.token) {
     return;
   }
-
-  service.countDatasetView(exploreDetail.data.id, user.token);
+  countDatasetView(exploreDetail.data.id, user.token);
```

Comment:
For consistency, I would name it `fetchDatasetCount` or similar... The current name doesn't give you any hint on what it does and it looks like more a component name to me. Thoughts?